pages/common/if.md

```diff
@@ -1,28 +1,36 @@
 # if
 
-> Simple shell conditional.
+> Performs conditional processing in shell scripts.
 > See also: `test`, `[`.
-> More information: <https://www.tcl.tk/man/tcl8.6.11/TclCmd/if.html>.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Conditional-Constructs>.
 
-- Execute two different commands based on a condition:
+- Execute the specified commands if the condition command's exit status is zero:
 
-`if {{command}}; then {{echo "true"}}; else {{echo "false"}}; fi`
+`if {{condition_command}}; then {{echo "Condition is true"}}; fi`
 
-- Check if a variable is defined:
+- Execute the specified commands if the condition command's exit status is nonzero:
 
-`if [[ -n "{{$VARIABLE}}" ]]; then {{echo "defined"}}; else {{echo "not defined"}}; fi`
+`if ! {{condition_command}}; then {{echo "Condition is true"}}; fi`
 
-- Check if a file exists:
+- Execute the first specified commands if the condition command's exit status is zero otherwise execute the second specified commands:
```

Comment:
Perhaps this will read better?
```suggestion
- Execute the first set of commands if the condition command's exit status is zero, otherwise execute the second set of commands:
```

Reply:
It seems that code snippet description is going to become very verbose. ;) Personally I like your suggestion but I have to think about when it's better to use smth like "list/set of" in command descriptions to write more standardized tldr pages. If anybody has any thoughts please tell me them, I am glad to hear them. ;)